snyk-monitor/templates/deployment.yaml

```diff
@@ -58,6 +58,8 @@ spec:
             value: {{ .Values.https_proxy }}
           - name: NO_PROXY
             value: {{ .Values.no_proxy }}
+          - name: LOG_LEVEL
+            value: {{ .Values.log_level }}
```

Comment:
I guess you need to include the same options to the helm templates [here](https://github.com/snyk/kubernetes-monitor/blob/ecca7f5ce26fc8926660871ed07896a3109199be/snyk-monitor/templates/deployment.yaml#L59) for `snyk-monitor/templates/deployment.yaml`

Reply:
Your link is pointing to the same place it's in PR, am I missing smth?